Formula/vault.rb

```diff
@@ -16,12 +16,24 @@ class Vault < Formula
     sha256 "550fda7af152696f0f8c7e1bbd228b7aa7802e6d9cb6f3a61e1fcbbc65462216" => :high_sierra
   end
 
-  depends_on "go@1.12" => :build
+  depends_on "go@1.13" => :build
   depends_on "gox" => :build
 
   def install
     ENV["GOPATH"] = buildpath
 
+    # GOPRIVATE should be removed when v1.4.1 is released.
```

Comment:
Can you reduce the comment to just this and the macports URL?

Reply:
done